episodes/04-data-structures-part2.Rmd

```diff
@@ -281,6 +283,59 @@ tail(gapminder_norway)
 
 To understand why R is giving us a warning when we try to add this row, let's learn a little more about factors.
 
+
+## Removing columns and rows in data frames
+
+To remove columns from a data frame, we can use the 'subset' function.
+This function allows us to remove columns using their names:
+
+```{r}
+life_expectancy <- subset(gapminder, select = -c(continent, pop, gdpPercap))
+head(life_expectancy)
+```
+
+We can also use a logical vector to achieve the same result. Make sure the
+vector's length match the number of columns in the data frame (to avoid vector
+recycling):
```

Comment:
maybe a link to the definition of vector recycling or writing it in plain language here?